src/bin/pg_autoctl/monitor.c

```diff
@@ -298,7 +298,7 @@ monitor_node_active(Monitor *monitor,
 		"$6::pgautofailover.replication_state, $7, $8, $9)";
 	int paramCount = 9;
 	Oid paramTypes[9] = { TEXTOID, TEXTOID, INT4OID, INT4OID,
-						  INT4OID, TEXTOID, BOOLOID, INT8OID, TEXTOID };
+						  INT4OID, TEXTOID, BOOLOID, TEXTOID, TEXTOID };
```

Comment:
Should we use `LSNOID` here? Given that we're using the text protocol then the server would cast the literal values in the `pg_lsn` data type for us automatically, after having found the server-side function with the matching signature, IIRC.